control_v2.proto

```diff
@@ -106,7 +106,7 @@ message ComponentUnitState {
   UnitType unit_type = 1;
   // ID of the unit in the component.
   string unit_id = 2;
-  // Current state.
+  // Current status.
```

Comment:
Why the change to status here? Probably no benefit to changing the contents of this file, minus the `go_package` above.

Reply:
thanks for noticing this is just a leftover from refactorings